.github/workflows/build_and_test.yml

```
@@ -79,7 +79,9 @@ jobs:
         # when a workflow is triggered from a forked repository.
         if: github.event.pull_request.head.repo.full_name == '3rdparty/eventuals'
         run: |
-          ${{ matrix.bazel-command }} build \
+          ${{ matrix.bazel-command }} \
+            --output_user_root=C:/bzl \
```

Need to make this:
```suggestion
            ${{ matrix.output-user_root }} \
```
So that we only do this for Windows!